deployment_scripts/deploy_python_as_dir.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+# deploy_python_as_dir.sh: deployment script for setting up a project in a frozen directory
+
+### config
+# pip: relative or absolute path to pip
+pip=
+
+# repo: remote url of the git repository
+repo=
+
+# developement or production: only used to create a link at the end
+environment=
+
+# any preliminary commands, exports, etc
+
+if [ ! $pip ] || [ ! $repo ]
+then
+    echo "Invalid config: ensure that pip and repo are populated"
+    exit 1
+fi
+### end config
+
+
+if [ $# == 0 ]
+then
+    echo "Usage: $0 tag/branch/master"
+    exit
+fi
+
+scriptpath=$(dirname $(readlink -f $0))
+PWD=$(pwd)
+
+cd "$scriptpath"
+git_element=$1
+
+if [ ! -d git_repo ]
+then
+    echo "Initialising local Git repo"
+    git init --bare --shared git_repo
+fi
+
+echo "Deploying $git_element from $repo"
+
+git --git-dir=git_repo fetch $repo "$git_element:$git_element"
```

Comment:
Will this work on an existing directory if it is frozen?

Reply:
I don't think it is required the same way it is for the deployment in `venv` (which won't deploy anything if the folder exists) but it is safer and cleaner.
I've added it